.github/workflows/.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.yml]
```

Comment:
Already covered by top-level configuration, so can be removed.

Reply:
thx